### DISCUSSION TITLE
BitcrazeVM Stuck On Startup

### DISCUSSION THREAD

**Discussion by Itai51** [bitcraze/discussions#1273] [Q&A]
Hello all,

I am attempting to run the BitcrazeVM on my Windows 11 PC. Initially I was able to access the virtual machine. Then I followed the tutorial to run the CFClient in the Windows command line. However, once I tried getting back into the VM, it gets stuck loading on the same line each time:
![image](https://github.com/bitcraze/discussions/assets/42220654/f2d68f93-97fd-44a4-bdc1-84ee7cbc744a)

What is causing this issue? I assume I accidentally corrupted a file somewhere down the road, but I cannot figure out how to fix it. I wouldn't mind completely cleaning my PC of all Crazyflie related stuff and restarting, but I am also unsure how to do that.

Any help is appreciated!

**Comment by ataffanel** [comment]
I am not sure what could be happening, but the way to reset the VM is to delete it from virtualbox and re-import the OVA to reinstall it.

**Reply by Itai51** [reply]
Thank you
